sql/core/src/main/resources/org/apache/spark/sql/execution/ui/static/spark-sql-viz.css

```diff
@@ -15,35 +15,35 @@
  * limitations under the License.
  */
 
-#plan-viz-graph .label {
+svg g.label {
```

Comment:
The inlined style to the exported SVG does not have the plan-viz-graph div as a container